backend/game_server/lib/helpers/vector.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Helpers.NotVector do defexception message: "One of arguments is not Vector" end
+
+
+
+defmodule Helpers.Vector do
+	alias :math, as: Math
+	alias Helpers.Vector, as: Vector
+	defstruct x: 0.0, y: 0.0
+
+
+	def distance(d1 = %Vector{},d2 = %Vector{}) do 
```

Comment:
Better version should be v1? or vector_1? This module is for operation od vectors